kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -221,73 +237,75 @@ class ResidualBasedBossakDisplacementScheme
         const double delta_time = r_current_process_info[DELTA_TIME];
 
         // Updating time derivatives (nodally for efficiency)
-        const auto it_node_begin = rModelPart.Nodes().begin();
+        if (rModelPart.Nodes().size() > 0) {
```

Comment:
This is the actual change (rest is tabulation, and previous one is typedef ->using)